examples/python/Quantinuum_arbitrary_angle_zz.py

```diff
@@ -1,13 +1,21 @@
+# <div style="text-align: center;">
+# <img src="https://assets-global.website-files.com/62b9d45fb3f64842a96c9686/62d84db4aeb2f6552f3a2f78_Quantinuum%20Logo__horizontal%20blue.svg" width="200" height="200" /></div>
+
 # # Arbitrary Angle ZZ Gates via pytket
 
-# This notebook contains a comparison of circuits with and without use of Quantinuum's native arbitrary-angle ZZ gate in `pytket`.
+# This notebook contains a comparison of circuits with and without use of Quantinuum's native arbitrary-angle ZZ gate in `pytket`. The circuit
+# primitive, the Quantum Fourier Transform (QFT) is constructed with `pytket`. The inverse QFT is an important primitive used in the [Phase
+# Estimation Algorithm (PEA)](https://en.wikipedia.org/wiki/Quantum_phase_estimation_algorithm). PEA is used to estimate the phase corresponding to the eigenvalue of a specified unitary.
```

Comment:
If you like you could link to my phase estimation notebook from here 

https://tket.quantinuum.com/examples/phase_estimation.html

Whatever you think is best